### DISCUSSION TITLE
Disable edit mode

### DISCUSSION THREAD

**Discussion by KizeRemi** [mdx-editor/editor#49] [General]
Hello, thanks for this library. 

Sometimes, I need to disable the edition for some users. So it's like a readonly mode.

When I check the DOM of the mdxeditor component, I see an attribute called `contenteditable` at true by default, but there is no related props. I can't update this attribute properly, it seems, I need to do it with an hack.

Any idea on how I can have a readonly mode with no hack?

Thank you :)

**Comment by petyosi** [comment]
This has been requested a few times, but so far my understanding was that people are confused that they need the editor to render its contents even in read only mode. 

From what I get though, you want to keep the "chrome" (toolbar, etc), and just prevent the users from changing the contents - similar to a read-only input. Let me know if I understand you correctly. This is possible to be implemented, but I need to look into it for more details.

**Comment by KizeRemi** [comment]
Thanks for your answser, 
In my case, the content is set by default with a template (depending of another input). A user with admin role can update the content with the toolbar or shortcut.
A non-admin user can only see the template and cannot editing him. I can hide the toolbar for now easily (only custom buttons will be available later).

**Comment by petyosi** [comment]
Thank you for the clarification. 

Just published v0.16.0 which includes a `readOnly` boolean prop on the component level.

**Comment by petyosi** [comment]
@KizeRemi I'm closing this, but let me know if there's more to it than what's released.
